### PR TITLE
add promethus and cors docs in getstarted + fix version number

### PR DESCRIPTION
**Pull request by sijingzhangzsj0604**

### CODE REVIEW

File: enterprise_versioned_docs/version-2.13.x/getstarted/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,529 @@
+---
+title: Monitoring with Prometheus and Grafana
+slug: /getstarted/prometheus-grafana
```
Comment (guoqqqi) [inline enterprise_versioned_docs/version-2.13.x/getstarted/prometheus.md:3]:
```suggestion
slug: /getstarted/prometheus
```

File: enterprise_versioned_sidebars/version-2.13.x-sidebars.json [hunk 2]
```diff
@@ -8,6 +8,14 @@
       "type": "doc",
       "id": "platform-architecture"
     },
+    {
+      "type": "category",
+      "label": "Get Started",
+      "items": [
+        "getstarted/prometheus-grafana",
```
Comment (guoqqqi) [inline enterprise_versioned_sidebars/version-2.13.x-sidebars.json:15]:
```suggestion
        "getstarted/prometheus",
```

Comment (guoqqqi) [reply enterprise_versioned_sidebars/version-2.13.x-sidebars.json:15]: 同时跟新侧边栏配置，移除需要删除的配置

**Comment by juzhiyuan** [comment]
https://api7-docs-git-fix-monitoring-guide-api7ai.vercel.app/enterprise/getstarted/prometheus

**Pull request closed without merging**